config/placeholder_replacement.go

```diff
@@ -186,3 +199,17 @@ func (v *placeholderReplacementVisitor) replacePackagePlaceholder(toReplace stri
 	}
 	return packageConfig.LocalDataDirectory(viamPackagesDir), nil
 }
+
+func (v *placeholderReplacementVisitor) replaceEnvironmentPlaceholder(toReplace string) (string, error) {
+	matches := environmentPlaceholderRegexp.FindStringSubmatch(toReplace)
+	if matches == nil {
+		return toReplace, errors.Errorf("failed to find substring matches for %q", toReplace)
+	}
+	variableName := matches[environmentPlaceholderRegexp.SubexpIndex("name")]
+	value, present := os.LookupEnv(variableName)
+	if !present {
+		return toReplace, errors.Errorf("no environment variable named %q for placeholder %q",
```

Comment:
super-nit: make named = defined

Reply:
:+1:

Reply:
So you're imagining `No environment variable defined HOME for placeholder ${environment.HOME}` instead of `No environment variable named HOME for placeholder ${environment.HOME}`

?

Reply:
hm actually doesn't flow great. Its just that env variables store system variables, so naming just feels a bit wrong but its a super nit. 

Sorry just leave.